clients/mobile/src/authentication/context/AuthContext.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { createContext, useState, useEffect, ReactNode } from 'react'
+import authService from '../services/authService'
+
+interface AuthContextProps {
+  accessToken: string | null;
+  idToken: string | null;
+  isAuthenticated: boolean;
+  loading: boolean;
+  logoutUser: () => Promise<void>;
+  setAccessToken: (token: string | null) => void;
+  setIdToken: (token: string | null) => void;
+  setIsAuthenticated: (isAuthenticated: boolean) => void;
+}
+
+const initialAuthContext = {
+  accessToken: null,
+  idToken: null,
+  isAuthenticated: false,
+  loading: true,
+  // eslint-disable-next-line @typescript-eslint/space-before-function-paren
```

Comment:
this shouldn't be here

Reply:
Removed.[Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/197/commits/cc280e8a37f9c1825144ff7ca528169d72298a0a)